src/scripts.js

```diff
@@ -37,6 +37,13 @@ const onPageLoad = () => {
   compareSteps();
 }
 
+const getData = () => {
+  return fetch("https://pacific-badlands-43237.herokuapp.com/api/v1/users")
+  .then(response => response.json());
+  .then(data => users = new UserRepository(data))
```

Comment:
I think something needs to be returned somehow. Somehow we want to return the JSON data